framework/web/AssetBundle.php

```diff
@@ -147,7 +152,10 @@ public function registerAssetFiles($view)
             $view->registerJsFile($manager->getAssetUrl($this, $js), $this->jsOptions);
         }
         foreach ($this->css as $css) {
-            $view->registerCssFile($manager->getAssetUrl($this, $css), $this->cssOptions);
+            $css = ArrayHelper::toArray($css);
```

Comment:
There is a [core code style guide](https://github.com/yiisoft/yii2/blob/master/docs/internals/core-code-style.md), but my notice comes from optimization thoughts and simplicity. When it can be done with PHP type-casting, we have no need to use something else.